solutions/33.js

```diff
@@ -0,0 +1,21 @@
+// Maricris Bonzo: @seemcat
+// create a function that takes in an array and a number
+// and returns an array without that number
+
+const solution = (arr, num) => {
+  let i = 0;
+  let newArr = []
+  while (i < arr.length) {
+    if (num !== arr[i]) {
+      newArr.push(arr[i]);
+      i++;
+    } else {
+      i++;
+    }
+  }
```

Comment:
@seemcat you can condense this as:
`   while (i < arr.length) {
     if (num !== arr[i]) {
       newArr.push(arr[i]);
     } 
	i++;
   }
`